src/main/java/core/basesyntax/figures/Rectangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.figures;
+
+public class Rectangle extends Figure {
+
+    private final double length;
+    private final double width;
+
+    public Rectangle(String color, double length, double width) {
+        super(color);
+        this.length = length;
+        this.width = width;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.round((length * width) * 10.0 / 10.0);
+    }
```

Comment:
Magic numbers should be avoided. If you need to round to a specific number of decimal places, define a constant with a descriptive name for the rounding factor.